src/track/track.h

```diff
@@ -308,6 +307,9 @@ class Track : public QObject {
     void markForMetadataExport();
     bool isMarkedForMetadataExport() const;
 
+    /// Prints track contents information, for debuging purtposes only
+    void printDebugInfo() const;
```

Comment:
This does not belong here. Please replace is with `QDebug operator<<(QDebug dbg, const Track& track)`. Example: `trackinfo.h`